### PR TITLE
ci: add automatically building and pushing of container images to ghcr.io

### PR DESCRIPTION
**Pull request by kkujawa-sumo**

### CODE REVIEW

**Review by sumo-drosiek — APPROVED** [review]
LGTM

File: .github/workflows/release_builds.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: Releases
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+-sumo-[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-sumo-[0-9]+-alpha.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-sumo-[0-9]+-beta.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-sumo-[0-9]+-rc.[0-9]+'
```
Comment (pmalek-sumo) [inline .github/workflows/release_builds.yml:9]: Do we really want to use `-sumo-[0-9]+` suffix for those images?

File: .github/workflows/release_builds.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+env:
+  SIDECAR_IMAGE: "ghcr.io/sumologic/tailing-sidecar"
+  OPERATOR_IMAGE: "ghcr.io/sumologic/tailing-sidecar-operator"
+  LATEST_TAG: "latest"
+
+jobs:
+  build:
```
Comment (pmalek-sumo) [inline .github/workflows/release_builds.yml:16]: 🤔 Maybe we can separate operator and sidecar build steps into 2 separate jobs? They'd then run in parallel (=faster) and finding what went wrong would be easier.

What do you think?

File: .github/workflows/release_builds.yml (hunk 1 above)
Comment (kkujawa-sumo) [reply .github/workflows/release_builds.yml:9]: I think that they are not needed.

**Review by pmalek-sumo — APPROVED** [review]
🤔 👍

**Pull request merged**